### PR TITLE
Bump version to 2.3.10

### PR DESCRIPTION
**Pull request by dmivankov**
There's some bug in old sdk used in some older images of 2.3.9.
Let's bump the version for easier consumption of the latest
transformations-cli with bumped sdk in.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -13,6 +13,9 @@ Changes are grouped as follows
 - `Security` in case of vulnerabilities.
 
 
+# [2.3.10] - 2023-11-22
+- TODO: write down the changes
```
Comment (mr-celo) [inline CHANGELOG:17]: Shouldn't we state here that `cognite-sdk` version was bumped?

Comment (dmivankov) [reply CHANGELOG:17]: Along with other things, ideally yes

**Comment by tugceozgur** [comment]
@dmivankov bumping version here as well needed AFAIK [cognite/transformations_cli/__init__.py]

**Comment by tugceozgur** [comment]
@dmivankov guess SequenceRows is deprecated on v7 instead using SequenceRowsDestination is recommended.

**Comment by dmivankov** [comment]
Confused why this didn't show up earlier :thinking: but will try to fixup

**Comment by dmivankov** [comment]
Ok, my bad, accidentally picked up sdk v7 along, removing now

**Comment by tugceozgur** [comment]
Hmm more errors :) 
Not sure about this one 
```
Credentials for test-cli-transform-oidc write failed to validate: DataModelsAcl got an unknown scope: UnknownScope(name='dataModelScope', data={'external_ids': ['test-space']}), expected an instance of one of: [DataModelsAcl.Scope.All, DataModelsAcl.Scope.SpaceID]. Example usage: DataModelsAcl(actions=[DataModelsAcl.Action.Read], scope=DataModelsAcl.Scope.All())

```

**Comment by tugceozgur** [comment]
Does it help to do pre-commit before push ? `mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

cognite/transformations_cli/commands/deploy/transformations_api.py:101: error: Name "SequenceRows" is not defined
tests/test_deploy.py:[41](https://github.com/cognitedata/transformations-cli/actions/runs/6955236066/job/18923610646?pr=204#step:5:42)5: error: Name "SequenceRows" is not defined
Found 2 errors in 2 files (checked 31 source files)

flake8...................................................................Failed
- hook id: flake8
- exit code: 1

cognite/transformations_cli/commands/deploy/transformations_api.py:14:1: F401 'cognite.client.data_classes.transformations.common.SequenceRowsDestination' imported but unused
cognite/transformations_cli/commands/deploy/transformations_api.py:101:16: F821 undefined name 'SequenceRows'
tests/test_deploy.py:15:1: F401 'cognite.client.data_classes.transformations.common.SequenceRowsDestination' imported but unused
tests/test_deploy.py:415:36: F821 undefined name 'SequenceRows'

version-match-check......................................................Passed`

**Comment by dmivankov** [comment]
Yep, that's something from v7 sdk. I accidentally added v7 bump into this PR, but now staying on v6 and checks pass

**Comment by tugceozgur** [comment]
Looks like all checks are passing, @mr-celo Do you have time to review? Thanks

**Review by mr-celo — APPROVED** [review]
Approved, assuming we are

**Pull request merged**